cmd/otk-gen-partition-table/main.go

```diff
@@ -14,26 +14,26 @@ import (
 	"github.com/osbuild/images/pkg/osbuild"
 )
 
-type OtkGenPartitionInput struct {
-	Options    *OtkPartOptions `json:"options"`
-	Partitions []*OtkPartition `json:"partitions"`
+type OtkPartInput struct {
+	Options    *OtkPartInputOptions     `json:"options"`
+	Partitions []*OtkPartInputPartition `json:"partitions"`
```

Comment:
I wonder if it wouldn't be cleaner to just drop the `OtkPart` prefix.  It's not like this is an importable module and there would be confusion for the types.  The context here is pretty clear I think and we could have `Input`, `InputOptions`, `InputPartitions` etc.

Reply:
Excellent suggestion!